### PR TITLE
Updated wordings and python sample code for long audio APIs

### PR DESCRIPTION
**Pull request by cherrycaol**

### CODE REVIEW

File: CustomVoice-API-Samples/Java/README.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
-## Custom Voice API
+# Custom Voice API
 
-This project contains samples of HTTP-based Microsoft Custom Neural Voice Batch Synthesis APIs.
+This project contains samples of HTTP-based Microsoft Neural Voice Long Audio APIs.
  
 ## Build the samples
 
-1. First, you must obtain a standard (not free) Speech API subscription key by following instructions in Microsoft Cognitive Services subscription.
-1. Make sure that you have Maven installed. Under the directory with the pom file, run the command `mvn assembly:assembly` to Compile the executable jar package.
```
Comment (xiaozhouz) [inline CustomVoice-API-Samples/Java/README.md:8]: why remove the line about "Maven"?

File: CustomVoice-API-Samples/Python/Readme.txt [hunk 2]
```diff
@@ -11,6 +12,9 @@ python voiceclient.py -h
 2.Get submitted voice synthesis list with filter and paged:
 python voiceclient.py --voicesynthesis -region centralindia -key your_key_here -status Succeeded -timestart "2020-01-01" -timeend "2020-01-23" -skip 0 -top 100
 
+2.Get submitted voice synthesis by ID:
```
Comment (xiaozhouz) [inline CustomVoice-API-Samples/Python/Readme.txt:15]: the  order is still 2. It is similar to get voice synthesis list, so you can say "Or get single sumbmitted voice synthesis by ID", don't mark 2.

File: CustomVoice-API-Samples/Python/voiceclient.py [hunk 3]
```diff
@@ -23,15 +24,15 @@
...
 parser.add_argument('-voiceId', action="store", nargs='+', dest="voiceId", help='the id of the voice which used to synthesis')
-parser.add_argument('-synthesisId', action="store", nargs='+', dest="synthesisId", help='the id of the voice synthesis which need to be deleted')
+parser.add_argument('-synthesisId', action="store", nargs='+', dest="synthesisId", help='the id of the voice synthesis which need to be queried or deleted')
 parser.add_argument('-locale', action="store", dest="locale", help='the locale information like zh-CN/en-US')
 parser.add_argument('-format', action="store", dest="format", default='riff-16khz-16bit-mono-pcm', help='the output audio format')
-parser.add_argument('-key', action="store", dest="key", required=True, help='the cris subscription key, like ff1eb62d06d34767bda0207acb1da7d7 ')
-parser.add_argument('-region', action="store", dest="region", required=True, help='the region information, could be centralindia, canadacentral or uksouth')
+parser.add_argument('-key', action="store", dest="key", required=True, help='the Speech service subscription key, like bb82464444c548dea4dce4376f3c7d26 ')
```
Comment (xiaozhouz) [inline CustomVoice-API-Samples/Python/voiceclient.py:30]: this is random key, not real key, right?

File: CustomVoice-API-Samples/Python/Readme.txt (hunk 2 above)
Comment (cherrycaol) [reply CustomVoice-API-Samples/Python/Readme.txt]: Fixed, thanks.

File: CustomVoice-API-Samples/Python/voiceclient.py (hunk 3 above)
Comment (cherrycaol) [reply CustomVoice-API-Samples/Python/voiceclient.py:30]: yes

File: CustomVoice-API-Samples/Java/README.md (hunk 1 above)
Comment (cherrycaol) [reply CustomVoice-API-Samples/Java/README.md:8]: Added it back, thanks.

**Pull request merged**